### ISSUE TITLE
Windows path errors

### ISSUE THREAD

**Issue by sckott** [RevolutionAnalytics/checkpoint#104]
@andrie Maybe you have some insight on this error I'm getting on Windows

From Stackoverflow it seems to be not a huge problem, and I've seen fixes that a user can do, but is there something within RRT that can be fixed, or not?

```
cygwin warning:
  MS-DOS style path detected: C:/Users/setup/Documents/scottsnewrepo/rrt/lib/x86_64-w64-mingw32/3.1.0/src/contrib/stringr_0.6.2.tar.gz
  Preferred POSIX equivalent is: /cygdrive/c/Users/setup/Documents/scottsnewrepo/rrt/lib/x86_64-w64-mingw32/3.1.0/src/contrib/stringr_0.6.2.tar.gz
  CYGWIN environment variable option "nodosfilewarning" turns off this warning.
  Consult the user's guide for more details about POSIX paths:
    http://cygwin.com/cygwin-ug-net/using.html#using-pathnames
```
